docs/topics/auth/default.txt

```diff
@@ -500,6 +500,40 @@ The login_required decorator
     :func:`django.contrib.admin.views.decorators.staff_member_required`
     decorator a useful alternative to ``login_required()``.
 
+The LoginRequired mixin
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using :doc:`class-based views </topics/class-based-views/index>`, you can
+get the functionality of the ``login_required`` decorator by using the following
+mixin:
+
+.. class:: django.contrib.auth.mixins.LoginRequiredMixin
+
+    .. versionadded:: 1.9
+
+    If a view is using this mixin, all requests by non-authenticated users
+    will be redirected to the login page.
+
+    If you set the ``login_url`` property on your view, this URL will be used
+    for the redirect. Otherwise, :setting:`settings.LOGIN_URL <LOGIN_URL>`
+    will be used. By default, this URL will be called with a query string parameter
+    ``"next"`` containing the URL the user should be redirected to after a successful
+    login. You can change this name by setting the ``redirect_field_name`` parameter
+    on your view. Example::
+
+        from django.contrib.auth.mixins import LoginRequiredMixin
+
+        class MyView(LoginRequiredMixin, View):
+            login_url = '/accounts/login/'
```

Comment:
Can you change that to something different than settings.LOGIN_URL, e.g. `/login/`